calico_kubernetes/calico_kubernetes.py

```diff
@@ -346,12 +403,6 @@ def _apply_tags(self, profile_name, pod):
         :return:
```

Comment:
Worth putting some more into the docstring here.  It should describe what the default set of tags are, and what the generated tags will be (maybe even with an example!)